Automattic/jetpack
# Related Posts: avoid squished images when img height isn't defined

Fixes #20028

#### Changes proposed in this Pull Request:

* The bug was introduced in #19845. 
* Not all themes define any kind of default height for images. Let's do it for our related posts images.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start from a site that's connected to WordPress.com, where you have related posts activated, and where you have enough posts to see related posts.
* In Jetpack > Settings > Traffic, ensure images are enabled for related posts.
* Activate the Twenty Eleven theme.
* Visit a post where you have related posts.
* See the squished images.
* Apply this patch.
* The images should now be displayed nicely.
* When switching to another theme such as Twenty Twenty One, the images should also be displayed nicely.
